### ISSUE TITLE
Model Causing Some Complaints

### ISSUE THREAD

**Issue by robertallenpayne** [nbrightproject/NBrightBuy#18]
I'm not sure if I'm just not setting up the models correctly.  But for example if I'm selling shirts, with models set for the different sizes (rather than options because we want to track inventory), if I set the retail price to $20 but leave the sale price at zero, I'm getting some complaints from customers.  The "Models" tab at the bottom of the product view shows $20 (struck through) followed by $0.

Some folks are seeing this and are saying "Hey, that should be free.  You're advertising it as free."

I have "solved" the issue by putting in the same price for Sale and for Retail.  But then the item is flagged as being on sale.

I've searched in the details.html for a way to remove the Models tab (the way I did the social media tab).  But I've had no success yet.  I don't really see a need to have the Models tab down below if it already displays on main product detail.  Any advice on how to remove it?

Or is there a more correct way to do what I'm trying to do?

Thanks!

**Comment by robertallenpayne** [comment]
Dang! Posted here and should have posted to CodePlex.  Moving thread.

**Comment by leedavi** [comment]
Are you using the "Classic" theme for the product list?  There's a known problem with the Cygnus theme.

**Comment by robertallenpayne** [comment]
Hi David,

I'm on my phone right now and the screen isn't large enough to let me select the Edit Page to find out for sure. But I'm confident that I am using classic. The reason I'm confident is that you had previously given me into on how to disable the sharing widget by editing the detail.html file in the Classic folder. Since the sharing widget is not showing up, that gives me the confidence to say I'm using classic. 

Thanks for your quick response! For now I'm just running a fake sale to mask the issue.

**Comment by robertallenpayne** [comment]
Hi David. 

After returning to the office this afternoon, I checked the ProductView module. It is using the Classic theme. 

Looking forward to hearing your thoughts.

**Comment by leedavi** [comment]
OK I think this might be down to an issue with some tests in the "display.html" template, although I thought it was all fixed in Classic.

The basic problem was to do with any tests ("IF") checking for a price should check a price of "0.00"

Might be an idea to download the latest version from DNN Store and install that to see if you still get the same problem.

**Comment by robertallenpayne** [comment]
Hi David. 

I went back to the DNN store and downloaded the most recent update (3.1.12 or similar - I was on 3.1.09). 

I did a fresh install on a test site. The $0 issue still shows up in the models tab. Definitely set to Classic. 

Am preparing to deploy an art gallery and the artist does not want to do the fake sale thing I usually do. 

Any luck tracking down the part of the details code I need to comment out to make it stop displaying the models tab at the bottom of the detail view?

Robert Allen Payne
612 805-8533
